linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_password_pam_unix_remember/ansible/shared.yml

```diff
@@ -5,14 +5,77 @@
 # disruption = medium
 {{{ ansible_instantiate_variables("var_password_pam_unix_remember") }}}
 
-- name: "Do not allow users to reuse recent passwords - system-auth (change)"
-  replace:
-    dest: /etc/pam.d/system-auth
-    regexp: '^(password\s+sufficient\s+pam_unix\.so\s.*remember\s*=\s*)(\S+)(.*)$'
-    replace: '\g<1>{{ var_password_pam_unix_remember }}\g<3>'
+- name: "{{{ rule_title }}}: set_fact"
+  set_fact:
+    {{% if product == "rhel8" -%}}
+    control: 'required'
+    {{%- else -%}}
+    control: 'requisite'
+    {{%- endif %}}
```

Comment:
I think these could just be build time jinja variables.
Once content is built an Ansible task is not expected to handle multiple platforms.

Or do you see value in having this customizable by an admin?